### ISSUE TITLE
Noobaa db pod crash on 100GB file upload

### ISSUE THREAD

**Issue by akmithal** [noobaa/noobaa-core#6691]
### Environment info
- NooBaa Version: master-20210801
- Platform: OCP 4.7

### Actual behavior
1. Noobaa db pod crashed due to IO

### Expected behavior
1. No pod should crash while doing IO

### Steps to reproduce
1. Started upload of 100GB file to 3 buckets from 3 different accounts.
2. SC is rebooted meanwhile

After few minutes, found upload failed as noobaa db pod crashed. 

```
[root@hpo-11 akshat]# s3u3 sync big-files/ s3://bucket-23/
...
...

upload failed: big-files/big_file_2 to s3://bucket-23/big_file_2 An error occurred (InternalError) when calling the UploadPart operation (reached max retries: 2): We encountered an internal error. Please try again.

```

```
[root@ocp-akshat-1-inf ~]# podn
NAME                                               READY   STATUS             RESTARTS   AGE
noobaa-core-0                                      1/1     Running            0          18h
noobaa-db-pg-0                                     0/1     CrashLoopBackOff   3          20h
noobaa-default-backing-store-noobaa-pod-b7304486   0/1     Terminating        0          14h
noobaa-endpoint-75fb65c979-hc2ch                   1/1     Running            0          3h4m
noobaa-operator-5c664b67c5-7nlgh                   1/1     Running            7          20h
```

### More information - Screenshots / Logs / Other output

```
[root@ocp-akshat-1-inf ~]# oc logs noobaa-db-pg-0 -p  
pg_ctl: another server might be running; trying to start server anyway
waiting for server to start....2021-08-03 08:22:59.914 GMT [26] LOG:  input in flex scanner failed at file "/var/lib/pgsql/data/userdata/postgresql.conf" line 1
2021-08-03 08:22:59.914 GMT [26] FATAL:  configuration file "/var/lib/pgsql/data/userdata/postgresql.conf" contains errors
 stopped waiting
pg_ctl: could not start server
Examine the log output.
```

**Comment by akmithal** [comment]
[must-gather-0308.tar.gz](https://github.com/noobaa/noobaa-core/files/6922497/must-gather-0308.tar.gz)

**Comment by akmithal** [comment]
Defect 6611 - https://github.com/noobaa/noobaa-core/issues/6611 is similar, so we can close this one and use the other one too.

**Comment by akmithal** [comment]
This issue seems to be fixed with ODF 4.9.4-7 build. Closing this defect.
